src/tooltip/src/Tooltip.js

```diff
@@ -9,114 +9,64 @@ import TooltipStateless from './TooltipStateless'
 
 let idCounter = 0
 
-export default class Tooltip extends PureComponent {
-  static propTypes = {
-    /**
-     * The appearance of the tooltip.
-     */
-    appearance: PropTypes.oneOf(['default', 'card']).isRequired,
-
-    /**
-     * The position the Popover is on.
-     */
-    position: PropTypes.oneOf([
-      Position.TOP,
-      Position.TOP_LEFT,
-      Position.TOP_RIGHT,
-      Position.BOTTOM,
-      Position.BOTTOM_LEFT,
-      Position.BOTTOM_RIGHT,
-      Position.LEFT,
-      Position.RIGHT
-    ]),
-
-    /**
-     * The content of the Popover.
-     */
-    content: PropTypes.node.isRequired,
-
-    /**
-     * Time in ms before hiding the Tooltip.
-     */
-    hideDelay: PropTypes.number.isRequired,
-
-    /**
-     * Time in ms before showing the Tooltip.
-     */
-    showDelay: PropTypes.number.isRequired,
-
-    /**
-     * When True, manually show the Tooltip.
-     */
-    isShown: PropTypes.bool,
+const Tooltip = memo(props => {
+  const {
+    appearance = 'default',
+    position = Position.BOTTOM,
```

Comment:
Any of these values that have defaults need to remove the `isRequired` from `propTypes` right?

Reply:
Yes. My bad